plugins/it.xsemantics.dsl/src/it/xsemantics/dsl/XsemanticsModelGenerator.xtend

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
```

Comment:
You can get trailing whitespace when you just use your own copyright header. Just try something that includes an empty line and you will see the trailing white space.

The copyright header support on your master, (that Jens implemented) has two issues: It does not support the format of Xsemantics itself and produces trailing whitespaces with your copyright header. That's why I chose to improve the Xsemantics project here and I thought you'd appreciate that. 

However, these improvements are just my optional fixes since the Eclipse Foundation has no constraints regarding these two issues. For the sake of getting all of the work we do for you done, I'll just revert the fixes of the two issues. Instead, we adapt our file checker regarding the trailing white spaces. Also, the N4JS copyright header is already consistent everywhere.